bucket/xpdf-tools.json

```diff
@@ -16,6 +16,13 @@
             "extract_dir": "xpdf-tools-win-4.03\\bin32"
         }
     },
+    "pre_install": [
+        "ensure \"$persist_dir\" | Out-Null",
+        "Get-ChildItem \"$persist_dir\\*\" -Include 'xpdfrc' | Copy-Item -Destination \"$dir\" -ErrorAction SilentlyContinue"
```

Comment:
no hardlink.

Reply:
But why? If you copy it rather than hardlink it, users may be confused that which configure file is used currently.

Reply:
I have found serval apps use hardlink than coping the configure files. such as potplayer, surmatrapdf, Ditto. They all work very well.

Reply:
99.9% of the manifests do not use hardlink, the apps you mentioned might just created hardlink for the folder by the scoop built-in method.

Reply:
for the case, just need to use the `xpdfrc` in the program directory according to the README of xpdf-tools. it will be copied correctly at every update/un/install.

Reply:
Nobody can create a hardlink for a folder.

Reply:
I believe you only need to place the file xpdfrc and the folder languagesupport in the $persist_dir
![image](https://user-images.githubusercontent.com/5350867/110775926-74bc3d00-829a-11eb-8c98-d04b77403315.png)

Reply:
😂 junction.

Reply:
Better than coping. scoop provides a better way.  Isn't it?

Reply:
i know what you mean, now. i never noticed that persist creates HARDLINK for the file(s.

however, the config file should be not mandatory, unless it is auto-generated by the app.
for the storage location, i think it should be consistent with main/tesseract-languages.

Reply:
reasons for not using hardlink:
· user habits ( accordin to docs/readme, may cz lose the config file
· unpredictable hardlink errors

reasons for not mv the LSP to the xpdf-tools persistent directory:
· external installation users ( if persist it, then you must install xpdf-tools first
· for other uses

Reply:
The language support packages may be updated seperately with xpdf-tools. So you adding them as seperate apps is a better way. So that user can update tthem via scoop when only the LSP is updated.

But I still think hardlinking xpdfrc is a better way. hardlinks mean two or more seperate same file. you can remove either one of them but not losing any data. you can access the data and/or hardlink it again via the other remaining hardlinks.

Reply:
> reasons for not using hardlink:
> 
> · user habits ( accordin to docs/readme, may cz lose the config file
> 
> · unpredictable hardlink errors
> 
> 
> 
> reasons for not mv the LSP to the xpdf-tools persistent directory:
> 
> · external installation users ( if persist it, then you must install xpdf-tools first
> 
> · for other uses 

Anyway. Thanks for taking the time to add these packages. That really help me a lot. I can install and update these packages via scoop after your PR is merged.